### ISSUE TITLE
`eslint-plugin-unicorn(explicit-length-check)` does not work

### ISSUE THREAD

**Issue by notramo** [oxc-project/oxc#1798]
`0 < myArray.length` is considered incorrect in favor of `myArray.length > 0`, however, I prefer using the `<` operator.
I guess it's implemented by checking only if the `.length` property is followed by an operator, but not if preceeded.

**Comment by IWANABETHATGUY** [comment]
According to the doc of `eslint-plugin-unicorn`,  `0 < myArray.length` is not allowed.
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/explicit-length-check.md#fail-1, 
![image](https://github.com/oxc-project/oxc/assets/17974631/9b9e1ff1-36dc-4bd9-91b2-220963105dee)

**Comment by notramo** [comment]
Does `oxlint` try to mirror the unicorn plugin exactly as it is?

**Comment by camc314** [comment]
Yes. we should also support the same options as unicorn plugin: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/explicit-length-check.md#options

When using `eslint-plugin-unicorn` does this also have an error?

**Comment by IWANABETHATGUY** [comment]
Here is the result of `eslint-plugin-unicorn': 
**code**: 
```js
let myArray = []
0 < myArray.length;

```
```
  2:1  error  Use `.length > 0` when checking length is not zero  unicorn/explicit-length-check

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

**Comment by IWANABETHATGUY** [comment]
You can just disable the rule if you don't like the lint result.

**Comment by notramo** [comment]
This lint is useful, however I prefer to use the less-than operator as it aligns better with left to right writing systems.

What about adding a `less-than` configuration option?

https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2009

**Comment by Boshen** [comment]
Closing as not planned, unless there are adjustments in eslint-plugin-unicorn.

Thank you for the bug report.
